libexec/python/utils.py

```diff
@@ -57,13 +56,13 @@ def add_http(url,use_https=True):
     prefix = "https://"
     if use_https == False:
         prefix="http://"
-    
+
     # Does the url have http?
-    if re.search('^http*',url) == None:
+    if not url.startswith('http'):
```

Comment:
I would recommend `http://` and/or `https://` to pedantically identify the valid URIs. While hostnames starting with 'http(s)' are not common, it doesn't hurt to be more specific.